UnityProject/Assets/Scripts/UI/Objects/Commons/Vending/GUI_Spawner.cs

```diff
@@ -31,14 +31,13 @@ protected override void InitServer()
 		SpawnedObjectList.OnObjectChange.AddListener( ( newObject, elementName, element ) =>
 		{
 			var netElement = (NetUIElement<string>) element;
-			switch ( elementName )
+			if (elementName == "MobName")
 			{
```

Comment:
imo switch  Is better here

Reply:
yes - it was unclear to me if codacy counted the default statement when counting for its fewer-than-three-cases style. It seems like it does so I will make it a switch again